modules/rules_swift/1.2.0/presubmit.yml

```diff
@@ -0,0 +1,17 @@
+matrix:
+  platform:
+  - centos7
+  - debian10
+  - ubuntu2004
+  - windows
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    build_targets:
+    - "//examples/xplatform/..."
```

Comment:
The targets have to be prefixed with the module name since the test setup uses an anonymous module as the root.

Reply:
Thanks! I'm also open to moving these to `bcr_test_module` at some point. Not sure exactly what are the guidelines, but I believe we'd like to add an example that uses bzlmod in `rules_swift` and then specify it here.?

Reply:
Yes, that would be ideal as it exercises the module from the outside - exactly what the BCR test should do.